test/performance/odp_sched_perf.c

```diff
@@ -241,7 +267,7 @@ static int create_queues(test_global_t *global)
 		return -1;
 	}
 
-	for (i = 0; i < ODP_THREAD_COUNT_MAX; i++)
+	for (i = 0; i < MAX_QUEUES; i++)
```

Comment:
Why not `tot_queue` here instead of `MAX_QUEUES`? Seems unnecessary to fill in unused elements.

Reply:
changed to tot_queue in v2.